CHANGELOG.md

```diff
@@ -1,108 +1,91 @@
+| Story | Add cypress status reporting while running headless                                                                                               | [3613][]                      |
+| Bug   | Gift options on Order Level (Commerce only)                                                                                                       | [3540][]                      |
+| Bug   | Parent product image thumbnail has been shown in mini cart instead of product itself                                                              | [3690][]                      |
+| Bug   | Long wishlist name length breaks layout                                                                                                           | [3706][]                      |
+| Bug   | Change default sort order on Search page to 'Relevance'                                                                                           | [3698][]                      |
+| Bug   | Wishlist page error when configured variant added                                                                                                 | [3703][]                     |
+| Bug   | Fix Cypress Page Builder tests                                                                                                                    | [3715][]                      |
+| Bug   | Dynamic Blocks GraphQL returns non-qualified blocks                                                                                        | [3678][], [3][]               |
+| Bug   | Remove virtual products extension                                                                                                                 | [3612][]                      |
+| Bug   | Possible wrong usage of useMemo                                                                                                        | [3660][]                      |
+| Bug   |  PWA multiple wish lists: storefront still showing create wishlist button when reaching the max number of allowed wish lists configured in Commerce    | [3668][]                      |
+| Bug   | Header - Logo dimensions overlap with main content                                                                                                | [3659][]                      |
+| Bug   | Click target for long-spanning tabs is inconsistent                                                                                          | [3676][]                      |
+| Bug   | Venia sample data personalized content not updating                                                                                               | [20][]                        |
 
 ## Documentation updates
 
--  **contactUs Mutation**: Added the [contactUs][] mutation topic to the GraphQL Developer Guide.
+-   Added [best practices documentation](https://developer.adobe.com/dreamweaver/guides/best-practices/index.html) for Customer Segments.
```

Comment:
Broken link

Reply:
It will resolve when we push the 12.3 work.

Reply:
PWA Studio best practices guide will be under the `dreamweaver` namespace?

Reply:
It was, back in 2004. Fixed.